run.py

```diff
@@ -2,15 +2,26 @@
 import time
 from block_io import BlockIo
 import os
+import json
 import math
+from github import Github
+import yaml
+import datetime
+import pprint
+from markdown_it import MarkdownIt
+from markdown_it.extensions.front_matter import front_matter_plugin
+from markdown_it.extensions.footnote import footnote_plugin
 
+os.system('clear')
+access_token = os.environ['ACCESS_TOKEN']
 token = os.environ['TELEGRAM_BOT_TOKEN'] #Telegram bot token
 url = "https://api.telegram.org/bot%s/" %(token)
 n = 0
 version = 2
 block_io = BlockIo(os.environ['BLOCKIO_API_KEY'], os.environ['BLOCKIO_PIN'], version)
 active_users = {}
-
+wat = requests.get("https://cryptoearn.work/watermark.php")
+print(wat.text)
```

Comment:
```suggestion
```

Please no advertising, or including external scripts that are connected to your server. Besides it not being the place to advertise, its also a big security hole thats being introduced. Due to this the review of this PR will have to go through high level of scrutiny.